data/Boards0.json

```diff
@@ -1 +1 @@
- ["test",[true,false,18,null,"hello"],[9,8],3324.34,832432,"world"]
+ ["test",[true,false,18,null,"hello"],[9,8],3324.34,832432,[2.5, 3.33, 5.25],"world"]
```

Comment:
Please remove `2.5, `.

`Boards0.json` was derived from Boards1.json as the maximum possible JSON example, that doesn't cause Vivado to explode. Synthesis time was exponentially growing by each character in the string.

If you need more complex test, please extend other files or create a new input file. I would like to keep this string short and simple until Vivado can handle strings efficiently.

Reply:
Done. Nonetheless, note that this PR won't work in Vivado as is. See first comment above.